test/specs/help/HelpService.spec.js

```diff
@@ -43,7 +56,8 @@ describe('ga_help_service', function() {
       });
 
       it('gets help from cache the 2nd time', function(done) {
-        $httpBackend.expectJSONP(frUrl).respond({columns: [], rows: []});
+        gaLang.lang = 'rm';
+        $httpBackend.expectJSONP(deUrl).respond({columns: [], rows: []});
```

Comment:
Why do you change this test?  the goal was to re-request the same help with the same language so 'fr'